### ISSUE TITLE
POMP weights for other CLIP variants

### ISSUE THREAD

**Issue by 100rab-S** [amazon-science/prompt-pretraining#13]
Hi @RenShuhuai-Andy,
I understand that the POMP weights are dependent on the model used for pre-training. 
So do you have weights for other variants of the CLIP model i.e. ViT-B/32 and ViT-L/14 backbone?

**Comment by RenShuhuai-Andy** [comment]
Hi, thanks for your interest.

Yes, you are correct that the POMP weights are dependent on the model used for pre-training. 

Currently, we don't have weights specifically for the ViT-B/32 and ViT-L/14 backbone variants of the CLIP model. However, I think it's a great suggestion and I would be happy to pre-train prompts for these variants. This process may take around one or two weeks to complete.

**Comment by 100rab-S** [comment]
@RenShuhuai-Andy Thanks for the reply. I'll be waiting for the pre-trained prompts for these variants.

**Comment by taoxinlily** [comment]
@RenShuhuai-Andy 
Thanks for yout great work! Are there pomp weights for the ViT-B/32 backbone variants of the CLIP model now? I'm looking forward to your reply!

**Comment by RenShuhuai-Andy** [comment]
Hi, sorry for the late reply. Since I have finished my internship in AWS, it takes me much time to re-prepare the datasets and re-train the prompt. 

The POMP weights for the **ViT-B/32** backbone are available in https://huggingface.co/ShuhuaiRen/POMP-ViT-Base-32/tree/main. The average cross-dataset accuracy for `vit_b32_ep5_randaug2_unc1000_16shots_nctx4_cscFalse_ctpend_seed42.pth.tar` and `vit_b32_ep20_randaug2_unc1000_16shots_nctx16_cscFalse_ctpend_seed42.pth.tar` is **62.0%** and **61.8%**, respectively.

The POMP weights for the **ViT-L/14** backbone are available in https://huggingface.co/ShuhuaiRen/POMP-ViT-Large-14/tree/main. The average cross-dataset accuracy for `vit_l14_ep5_randaug2_unc1000_16shots_nctx4_cscFalse_ctpend_seed42.pth.tar` and `vit_l14_ep20_randaug2_unc1000_16shots_nctx16_cscFalse_ctpend_seed42.pth.tar` is **72.4%** and **71.9%**, respectively.

**Comment by 100rab-S** [comment]
Thanks @RenShuhuai-Andy!
